src/Uno.UI/UI/Xaml/Documents/Run.skia.cs

```diff
@@ -275,5 +282,13 @@ static List<TextFormatting.GlyphInfo> GetGlyphs(HarfBuzzSharp.Buffer buffer, int
 		}
 
 		partial void InvalidateSegmentsPartial() => _segments = null;
+
+		private static Func<SKFont, float> _measureTab =
+			((Func<SKFont, float>?)(font1 =>
+			{
+				var space = ' ';
+				return font1.MeasureText(new ReadOnlySpan<ushort>(space)) * SpacesPerTab;
```

Comment:
(Verify if it works locally first)

```suggestion
				ReadOnlySpan<ushort> space = [' '];
				return font1.MeasureText(space) * SpacesPerTab;
```